language/rules-es5.json

```diff
@@ -1,5 +1,6 @@
 {
 	"rules": {
-		"es-x/no-symbol-prototype-description": "off"
+		"es-x/no-symbol-prototype-description": "off",
+		"es-x/no-resizable-and-growable-arraybuffers": "off"
```

Comment:
Should this not be error for es5 and off for es6?

Reply:
.resize is not available in all versions < ES2020 or something, so upstream sets it to "error" in those, however we need to turn it off, so we set it "off" in our lowest-supported language (ES5) so it propagates through to all languages.